homeassistant/components/assist_pipeline/pipeline.py

```diff
@@ -670,6 +680,17 @@ async def wake_word_detection(
         if result is None:
             wake_word_output: dict[str, Any] = {}
         else:
+            # Avoid duplicate detections by checking cooldown
+            last_wake_up = self.hass.data.get(DATA_LAST_WAKE_UP)
+            if last_wake_up is not None:
+                sec_since_last_wake_up = time.monotonic() - last_wake_up
+                if sec_since_last_wake_up < wake_word_settings.cooldown_seconds:
+                    _LOGGER.debug("Duplicate wake word detection occurred")
+                    raise WakeWordDetectionAborted
```

Comment:
Should we return an error code to indicate to the satellite that a wake word was found but it was a duplicate?

Reply:
We can do that maybe in the future.